projects/testing-library/tests/issues/issue-280.spec.ts

```diff
@@ -0,0 +1,55 @@
+import {Component, NgModule} from '@angular/core';
+import {render, screen} from '../../src/public_api';
+import {RouterModule, Routes} from "@angular/router";
+import {RouterTestingModule} from "@angular/router/testing";
+import {click} from "@testing-library/user-event/dist/click";
```

Comment:
```suggestion
import userEvent from "@testing-library/user-event";
```

And then in the test it becomes:

```
userEvent.click(...);
```

Reply:
I don't see benefit making it explicit and longer, but will change.